python/minorminer.py

```diff
@@ -0,0 +1,42 @@
+from minorminer_c import miner, VARORDER, find_embedding as __find_embedding
+
+
+def find_embedding(S, T,
+                   max_no_improvement=10,
+                   random_seed=None,
+                   timeout=1000,
+                   max_beta=None,
+                   tries=10,
+                   inner_rounds=None,
+                   chainlength_patience=10,
+                   max_fill=None,
+                   threads=1,
+                   return_overlap=False,
+                   skip_initialization=False,
+                   verbose=0,
+                   initial_chains=(),
+                   fixed_chains=(),
+                   restrict_chains=(),
+                   suspend_chains=()
+                   ):
+    return __find_embedding(S, T,
+                            max_no_improvement=max_no_improvement,
+                            random_seed=random_seed,
+                            timeout=timeout,
+                            max_beta=max_beta,
+                            tries=tries,
+                            inner_rounds=inner_rounds,
+                            chainlength_patience=chainlength_patience,
+                            max_fill=max_fill,
+                            threads=threads,
+                            return_overlap=return_overlap,
+                            skip_initialization=skip_initialization,
+                            verbose=verbose,
+                            initial_chains=initial_chains,
+                            fixed_chains=fixed_chains,
+                            restrict_chains=restrict_chains,
+                            suspend_chains=suspend_chains,
+                            )
+
+
+find_embedding.__doc__ = __find_embedding.__doc__
```

Comment:
I think https://docs.python.org/2/library/functools.html#functools.wraps will give you this plus cover some other edge cases.

Reply:
Is that nicer?  I find the following to be surprising/misleading:

`In [2]: minorminer.find_embedding
Out[2]: <function minorminer_c.find_embedding>`

Reply:
Fair point. Though it could be argued that for practical purposes it is that function in the `minorminer_c` module, you are functionally just importing it into the `minorminer` namespace. I am honestly unsure which is more surprising.

Reply:
I made the change that produced that line and the code looks cleaner, so I think I'll just ship it as is despite the wart